homeassistant/components/swiss_public_transport/const.py

```diff
@@ -9,12 +9,19 @@
 CONF_VIA: Final = "via"
 
 DEFAULT_NAME = "Next Destination"
+DEFAULT_UPDATE_TIME = 90
 
 MAX_VIA = 5
 SENSOR_CONNECTIONS_COUNT = 3
+SENSOR_CONNECTIONS_MAX = 15
```

Comment:
It's a bit confusing to prefix with `SENSOR_` when the new service is no longer an entity service, Maybe just `CONNECTIONS_MAX` or `MAX_CONNECTIONS`?

Reply:
yeah, makes sense